templates/base.html

```diff
@@ -151,12 +151,12 @@
     <!-- End Google Tag Manager (noscript) -->
 
     <!-- Google tag (gtag.js) -->
-        <script async src="https://www.googletagmanager.com/gtag/js?id=G-5S6RP1RFZ2"></script>
+        <script async src="https://www.googletagmanager.com/gtag/js?id={{ GOOGLE_GTAG }}"></script>
         <script>
           window.dataLayer = window.dataLayer || [];
           function gtag(){dataLayer.push(arguments);}
           gtag('js', new Date());
-          gtag('config', 'G-5S6RP1RFZ2');
```

Comment:
was this a valid value?  Is it a problem that this has been committed previously?